app/namespaces/test_adjustment/test_adjustment_service.py

```diff
@@ -233,19 +240,32 @@ def get_adjusted_estimate(self, test_adj, ind_se, ind_sp, ind_se_n, ind_sp_n,
                 upper = None
             else:
                 print('ADJUSTING ESTIMATE', adj_type)
-                model_params = dict(
-                    n_prev_obs=int(denominator_value),
-                    y_prev_obs=int(
-                        serum_pos_prevalence * denominator_value),
-                    n_se=int(output_se_n),
-                    y_se=int(output_se_n * se),
-                    n_sp=int(output_sp_n),
-                    y_sp=int(output_sp_n * sp)
-                )
-                lower, adj_prev, upper = self.pystan_adjust(model_params)
-                if pd.isna(adj_prev):
-                    print(f'FAILED TO ADJUST ESTIMATE')
-
+                # Check bounds of se, se_n, sp, sp_n
+                if se and not (se <= 1 and se >= 0.005):
+                    logging.error("se is not between 0.005 and 1")
+                    lower, adj_prev, upper = None, None, None
+                elif sp and not (sp <= 1 and sp >= 0.005):
+                    logging.error("sp is not between 0.005 and 1")
+                    lower, adj_prev, upper = None, None, None
+                elif output_se_n and output_se_n <= 1:
+                    logging.error("se_n is not between 1 and 100")
+                    lower, adj_prev, upper = None, None, None
+                elif output_sp_n and output_sp_n <= 1:
+                    logging.error("sp_n is not between 1 and 100")
```

Comment:
The log here implies that sp_n should be between 1 and 100, although we seem to only care about it being less than 1. Can we update the logs to be consistent with what we actually need to validate

Reply:
done